### ISSUE TITLE
missing apostrophes in PGN data (usually in the Opening PGN field)

### ISSUE THREAD

**Issue by snuggles4553** [lukasmonk/lucaschessR2#180]
It appears Lucas Chess doesn't include apostrophes in the `Opening` PGN field (or perhaps they get removed somehow).

Hence, we see things like this in PGN data generated by Lucas Chess:

- `[Opening "King Indian Attack"]`
- `[Opening "King Indian Defence: Orthodox, Aronin-Taimanov, Bayonet Attack"]`
- `[Opening "King Indian Defence: Petrosian System, Stein Variation"]`
- `[Opening "Queen Gambit Declined Semi-Slav: 5...Nd7"]`
- `[Opening "Queen Gambit Declined Slav: 4.e3"]`
- `[Opening "Queen Gambit Declined Slav: 5.e3 (Alekhine Variation)"]`
- `[Opening "Queen Gambit Declined: 4.Nf3"]`

Expected would be: `King's Indian Attack`, and `Queen's Gambit`, and so on.

This may apply to other openings that have apostrophes, such as the following (this list may be incomplete):

- Alapin's Opening
- Alekhine's Defence
- Anderssen's Opening
- Bird's Opening
- Grob's Attack
- King's Fianchetto Opening
- King's Indian Attack
- King's Indian Defence
- King's Pawn Game
- Owen's Defence
- Petrov's Defence
- Queen's Gambit
- Queen's Gambit Accepted
- Queen's Gambit Declined
- Queen's Pawn Game

**Comment by lukasmonk** [comment]
Thanks.
Done.
To the next update.

**Comment by lukasmonk** [comment]
Yes, I thought I had changed them all, there are 900 changes, I'll check again.

**Comment by lukasmonk** [comment]
Please, check again in the next update.
